src/Component/Board/Board.js

```diff
@@ -3,18 +3,27 @@ import React from 'react'
 // Board :: Props -> React.Component
 export default ({
   addCard,
-  isLoading,
-  cards
+  cards,
+  cardTitle,
+  isBlankCardVisible,
+  saveCard,
+  setBlankCardTitle,
 }) =>
-  <main className="myBoard">
+  <main className="my-board">
+
+    <input
+      className={isBlankCardVisible ? "new-card" : "new-card hidden"}
+      placeholder="What needs to be done?"
+      value={cardTitle}
+      onChange={e => setBlankCardTitle(e.target.value)}
+      onKeyUp={e => e.keyCode === 13 && saveCard(e.target.value)}
```

Comment:
Wouldn't be easier to just create a form and let the form handle the enter key (using onSubmit and a submit button)?
It's just a question, the way you did it works anyway.